health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/UpdateChecklist.js

```
@@ -284,14 +325,22 @@ const UpdateChecklist = () => {
     let uniqueLocal;
 
     const payloadData = (data) => {
+
+        data.forEach((question) => {
+            if (question.type.code === "Short Answer") {
+                question.type.code = "String";
+                delete question.options;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)